models/calendar.yml

```diff
@@ -319,10 +319,17 @@
         title: Introductions
         emoji: 👋
 
-    - 7:45 PM:
+    - 7:45 PM:  
         type: Talk
         emoji: 🎨
-        title: TBD
+        title: 'Tech Feminism 101: Why We Should All Be Feminists'
+        description: '2016 has been a good year for Feminism and our 
```

Comment:
Sure. I'll look at it later today.
On Tue, Aug 16, 2016 at 10:41 AM Billy Roh <notifications@github.com> wrote:

> In models/calendar.yml
> <https://github.com/wafflejs/wafflejs.github.io/pull/189#discussion_r74983377>
> :
>
> >          type: Talk
> >          emoji: 🎨
> > -        title: TBD
> > +        title: 'Tech Feminism 101: Why We Should All Be Feminists'
> > +        description: '2016 has been a good year for Feminism and our
>
> @ainsleywagon <https://github.com/ainsleywagon> Let me know if you'd like
> help with this!
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/wafflejs/wafflejs.github.io/pull/189/files/b6f959616b120a00d484eadb1d763bc45a82040e#r74983377>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABuT3Z_5BED0ZgO2IDj3TriTCaeyyxlaks5qgfZUgaJpZM4JimfE>
> .
>